guides/source/active_job_basics.md

```diff
@@ -131,77 +119,274 @@ That's it!
+
+```bash
+$ bin/rails db:migrate:queue
+```
+
+TIP: You can find the default generated schema for the `queue` database in
+`db/queue_schema.rb`. They will contain tables like
+`solid_queue_ready_executions`, `solid_queue_scheduled_executions`, and more.
+
+Finally, to start the queue and start processing jobs you can run:
+
+```bash
+bin/jobs start
+```
+
+#### Production
+
+Solid Queue is already configured for the production environment. If you open `config/environments/production.rb`, you will see the following:
+
+```ruby
+# config/environments/production.rb
+# Replace the default in-process and non-durable queuing backend for Active Job.
+config.active_job.queue_adapter = :solid_queue
+config.solid_queue.connects_to = { database: { writing: :queue } }
+```
+
+Additionally, the database connection for the `queue` database is configured in `config/database.yml`:
+
+```yaml
+# config/database.yml
+# Store production database in the storage/ directory, which by default
+# is mounted as a persistent Docker volume in config/deploy.yml.
+production:
+  primary:
+    <<: *default
+    database: storage/production.sqlite3
+  queue:
+    <<: *default
+    database: storage/production_queue.sqlite3
+    migrations_paths: db/queue_migrate
+```
+
+### Configuration
+
+The configuration options for Solid Queue are defined  in `config/queue.yml`. Here is an example of the default configuration:
+
+```yaml
+default: &default
+  dispatchers:
+    - polling_interval: 1
+      batch_size: 500
+  workers:
+    - queues: "*"
+      threads: 3
+      processes: <%= ENV.fetch("JOB_CONCURRENCY", 1) %>
+      polling_interval: 0.1
+```
+
+In order to understand the configuration options for Solid Queue, you must understand the different types of roles:
+
+- **Dispatchers**: They select jobs scheduled to run for the future. When it's time for these jobs to run, dispatchers move them from the `solid_queue_scheduled_executions` table to the `solid_queue_ready_executions` table so workers can pick them up. They also manage concurrency-related maintenance.
+- **Workers**: They pick up jobs that are ready to run. These jobs are taken from the `solid_queue_ready_executions` table.
+- **Scheduler**: This takes care of recurring tasks, adding jobs to the queue when they're due.
+- **Supervisor**: It oversees the whole system, managing workers and dispatchers. It starts and stops them as needed, monitors their health, and ensures everything runs smoothly.
+
+Everything is optional in the `config/queue.yml`. If no configuration at all is
+provided, Solid Queue will run with one dispatcher and one worker with default
+settings. Below are some of the configuration options you can set in `config/queue.yml`:
+
+| **Option**                           | **Description**                                                                                     | **Default Value**                             |
```

Comment:
This table feels very useful (to anye developer trying to implement Solid Queue). On the other hand, if any of these values change in the future, it'll become outdated. But I think we should keep it, as it's useful.

Reply:
Ok, we can keep an eye on it to see how often these get updated and then change course if need be - but I agree that I think its useful to have here.